src/doom/am_map.c

```diff
@@ -674,6 +680,11 @@ void AM_LevelInit(boolean reinit)
                 color_shades[color * NUMSHADES + shade] = colormaps[shade_index[shade]];
             }
         }
+		
+        secretwallcolors = V_GetPaletteIndex(W_CacheLumpName("PLAYPAL", PU_CACHE),
```

Comment:
@fabiangreffrath, how do you think, wouldn't it be better to lock "PLAYPAL" here in `PU_STATIC`, and later unlock it with `W_ReleaseLumpName`? This should be slightly more optimal, since this way we don't need to hit `W_CacheLumpName` twice. 
Same as done in https://github.com/fabiangreffrath/crispy-doom/blob/master/src/doom/r_data.c#L1076

Reply:
Yes, good idea. Thank you.